skaffold.yaml

```diff
@@ -1,4 +1,16 @@
-apiVersion: skaffold/v4beta2
+.im-group: &chart-config
```

Comment:
Why is the key different from the anchor?

Especially with `.im-group-prod: &service-account-config` we loose that the anchor references prod values. This is a bit confusing to me 🤔

Reply:
Sure, I can make the key and anchor the same, just looked weird to me. 🤷